docs/authenticate.md

```diff
@@ -40,18 +40,22 @@ You can use an access token to authenticate to the HERE platform and start worki
    import { UserAuth, requestToken } from "@here/olp-sdk-authentication";
    ```
 
-5. Create the `UserAuth` instance in one of the following ways:
+5. Create the `UserAuth` instance and, if needed, specify the token expiration time in one of the following ways:
 
-   - For token authentication, specify the `tokenRequester` method and your credentials.
+    > #### Note
+    > The default token expiration time must be equal to or more than zero. Ignored if it is zero or greater than the default expiration time supported by the access token endpoint. The default token expiration time is 3600 seconds (one hour).
+
+   - For token authentication, specify the `tokenRequester` method, your credentials, and the token expiration time.
 
      ```typescript
      const userAuth = new UserAuth({
        tokenRequester: requestToken,
        credentials: credentials,
+       expiresIn?: number;
      });
      ```
 
-   - To authenticate with local authorization, specify the environment in which you work, your credentials, and the `tokenRequester` method.
+   - To authenticate with local authorization, specify the environment in which you work, your credentials, the `tokenRequester` method, and the token expiration time.
```

Comment:
it's optional parameter, I'm not sure if we need to add that there

Reply:
Specified that it's optional in the note.